src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -244,8 +244,7 @@
 		<property name="stateService" ref="stateService" />
     </bean>
 
-	<bean id="batchFileDirectoryService" parent="batchFileDirectoryService-parentBean"/>
-	<bean id="batchFileDirectoryService-parentBean" abstract="true" class="edu.cornell.kfs.sys.service.impl.BatchFileDirectoryServiceImpl" />
+	<bean id="batchFileDirectoryService" class="edu.cornell.kfs.sys.service.impl.CuBatchFileDirectoryServiceImpl" />
```

Comment:
This should ideally be updated to extend from "batchFileDirectoryService-parentBean". The parent bean doesn't have any special property setup yet, but if that ends up changing, we'll need to make sure the bean inheritance gets straightened out.